src/itemChoice/style.ts

```diff
@@ -2,7 +2,7 @@ import css from 'styled-jsx/css'
 import { color, font, space } from '_utils/branding'
 
 const separatorHeight = '16px'
-const intValue = (value:string) => parseInt(value, 10)
+const intValue = (value: string) => parseInt(value, 10)
```

Comment:
why do you need this?

Reply:
Good question, I don't this just prettier reformatting